include/Chess.h

```diff
@@ -1,40 +1,40 @@
-#pragma once
-#include <iostream>
-#include <Windows.h>
-#include <string>
-
-using std::cout;
-using std::cin; 
-using std::endl;
-using std::string; 
-
-const int _SIZE = 21;
-
-class Chess {
-	unsigned char m_board[_SIZE][_SIZE] = { 0 };
-	bool m_turn = true;
-	string m_boardString;
-	string m_input;
-	string m_msg = "\n";
-	string m_errorMsg = "\n";
-	int m_codeResponse;
-
-	void clear() const;
-	void setFrames();
-	void setPieces();
-	void show() const;
-	void displayBoard() const;
-	void showAskInput() const;
-	bool isSame() const;
-	bool isValid() const;
-	bool isExit() const;
-	void excute();
-	void doTurn();
-
-public:
-	Chess(const string& start = "RNBQKBNRPPPPPPPP################################pppppppprnbqkbnr");
-	Chess(const Chess&)=delete;
-	Chess& operator=(const Chess&) = delete;
-	string getInput();
-	void setCodeResponse(int codeResponse);
+#pragma once
+#include <iostream>
+#include <Windows.h>
+#include <string>
+
+using std::cout;
+using std::cin; 
+using std::endl;
+using std::string; 
+
+const int _SIZE = 21;
+
+class Chess {
+	unsigned char m_board[_SIZE][_SIZE] = { 0 };
+	bool m_turn = true;
+	string m_boardString;
+	string m_input;
+	string m_msg = "\n";
+	string m_errorMsg = "\n";
+	int m_codeResponse;
+
+	void clear() const;
+	void setFrames();
+	void setPieces();
+	void show() const;
+	void displayBoard() const;
+	void showAskInput() const;
+	bool isSame() const;
+	bool isValid() const;
+	bool isExit() const;
+	void excute();
+	void doTurn();
+
+public:
+	Chess(const string& start = "RNBQKBNRPPPPPPPP################################pppppppprnbqkbnr");
+	Chess(const Chess&)=delete;
+	Chess& operator=(const Chess&) = delete;
+	string getInput();
+	void setCodeResponse(int codeResponse);

```

Comment:
the class Chess has member functions that modify the internal state of the class, such as setCodeResponse(), setFrames(), setPieces(), etc. However, these functions are not marked as const, even though they do not modify the logical state of the object. This can make the code less clear and may also prevent the object from being used in certain contexts.